src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -506,6 +506,14 @@ auto_flash(std::shared_ptr<xrt_core::device> & workingDevice, const std::string&
     }
     image_path = available_shells.front().second.get<std::string>("file");
   }
+  else if(boost::filesystem::exists(boost::filesystem::path(image))) {
```

Comment:
This change addresses the flashing a pdi issue. If the provided `image` is a valid path, and not a base name, we go ahead and flash it